schema/dereferenced-release-schema.json

```diff
@@ -26,68 +26,53 @@
       "omitWhenMerged": true
     },
     "publisher": {
-      "title": "Publisher",
-      "description": "The original publisher of this release.",
+      "title": "Identifier",
+      "description": "A unique identifier for a party (organization).",
```

Comment:
Hmm, I think my intention is for the title and description of the referring object to be preserved. I've solved this elsewhere, so I can have a look at the manage.py command.

Reply:
I've now added a commit.

Reply:
Hmm now, all the deprecations are in the party object too, is that correct/expected?

Reply:
They're expected in `buyer`, but not in `parties`. Where are you seeing it?

Reply:
Oh you are right, they are only in `buyer`, etc and not in `parties` itself